app/views/Project/Tagging/Sources/SourcesTable/EntryList/EntryItem/EntryVerification/EntryUnverifyCommentModal/index.tsx

```diff
@@ -145,19 +129,19 @@ function EntryUnverifyCommentModal(props: Props) {
                 label="Comment"
                 value={value.text}
                 onChange={setFieldValue}
+                error={error?.text}
                 rows={3}
                 autoFocus
             />
-            <MultiSelectInput
-                className={styles.input}
+            <ProjectMembersMultiSelectInput
                 name="mentionedUsers"
                 label="Assignees"
                 value={value.mentionedUsers}
+                projectId={projectId}
                 onChange={setFieldValue}
-                options={projectMembersResponse?.results}
-                keySelector={memberKeySelector}
-                labelSelector={memberNameSelector}
-                disabled={projectMembersPending}
+                options={members}
+                onOptionsChange={setMembers}
+                error={error?.mentionedUsers?.toString()}
```

Comment:
same as above

Reply:
I am referring to the comment I made above.
The error sent here should be calculated from a helper function.